src/Jackett.Common/Indexers/Nordicbits.cs

```diff
@@ -674,35 +666,35 @@ private void LatencyNow()
         /// <summary>
         /// Find torrent rows in search pages
         /// </summary>
-        /// <returns>JQuery Object</returns>
-        private CQ FindTorrentRows()
+        /// <returns>List of rows</returns>
+        private IHtmlCollection<IElement> FindTorrentRows(IHtmlDocument dom)
         {
             var defaultTheme = new[] { "/templates/1/", "/templates/2/", "/templates/3/", "/templates/4/", "/templates/5/", "/templates/6/", "/templates/11/", "/templates/12/" };
             var oldV2 = new[] { "/templates/7/", "/templates/8/", "/templates/9/", "/templates/10/" };
             var xmas = new[] { "/templates/14/" };
 
-            if (xmas.Any(_fDom.Document.Body.InnerHTML.Contains))
+            if (xmas.Any(dom.QuerySelector("body").InnerHtml.Contains))
```

Comment:
Use `.Body` property of IHtmlDocument here and below
```suggestion
            if (xmas.Any(dom.Body.InnerHtml.Contains))
```